Cakes-Co/index.html

```diff
@@ -7,10 +7,108 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
+     <link rel="stylesheet" href="style.css">
+     <link href="https://fonts.googleapis.com/css2?family=Pacifico&family=Poppins:wght@400;700&display=swap" rel="stylesheet">
 </head>
-
 <body>
     <!-- Add your markup here -->
+
+
+
+
+   
+<header class="header">
+    <div class="gray-cover"></div>
+    <h1 class="header-text"> The best cakes in the town <br>delivered to your door</h1>
+</header>
+
+
+
+<nav class="navbar">
```

Comment:
Same here, no need to duplicate the class `navbar`, we can apply the styles on the `nav` tag directly.